__tests__/app.test.js

```diff
@@ -301,9 +301,29 @@ describe('8-patch', () => {
           title:"Ultimate Werewolf",
           review_body:'We couldn\'t find the werewolf!'
         })
-      )
+    )
 
     })
 
   })
 })
+describe('9. GET /api/categories', () => {
```

Comment:
Be careful when copying and pasting... I think this is supposed to be GET /api/users? And the text description needs changing too